locales/en.yml

```diff
@@ -6,7 +6,7 @@ en:
           wtf:
             syntax: wtf is <term>?
             desc: Get the description of <term>
-          define:
+          desc:
```

Comment:
Weeeirrd.  So line #14 in the handler is what references this:

`t('help.define.syntax') => t('help.define.desc')`

So I _think_ line 11 actually needs to be updated instead, to follow the YAML tree?